tests/tape/interfaces/test_qnode_autograd.py

```diff
@@ -38,7 +37,7 @@ def test_execution_no_interface(self, dev_name, diff_method):
         def circuit(a):
             qml.RY(a, wires=0)
             qml.RX(0.2, wires=0)
-            return expval(qml.PauliZ(0))
+            return qml.expval(qml.PauliZ(0))

```

Comment:
Wouldn't these be now changed to the real measurement functions by having e.g. `qml.expval` and the tape is not enabled?

Reply:
Here the tape is enabled, since we are explicitly importing the tape qnode

Reply:
Oh, I see! How does the import enable the tape? Could it not lead to confusion? :thinking:

Reply:
Oh, there are two ways to use the tape QNode:

* Directly use `@qml.tape.qnode` or `@qml.tape.QNode`. This is recommended for testing, or if you want to write a script with both old and new QNodes.

* Use `qml.enable_tape()`. This function 'monkeypatches' PennyLane, so that calls to `@qml.qnode` instead will give you `@qml.tape.qnode`! It's similar to TensorFlow's `tf.enable_eager_execution()` function. This is recommended more for users.

Reply:
Cool, thanks! :slightly_smiling_face: The missing piece was the decorator